test/lib/usd/translators/UsdImportCustomConverterTest/UsdImportCustomConverterTest.usda

```diff
@@ -0,0 +1,182 @@
+#usda 1.0
+(
+    defaultPrim = "pCube1"
+    metersPerUnit = 0.01
+    upAxis = "Y"
+)
+
```

Comment:
Provides a mix of materials to test import scenarios (display color, preview surface, maya dump).